xml/System.Security.Cryptography.Pkcs/Pkcs12CertBag.xml

```diff
@@ -68,9 +72,18 @@ Note that for X.509 public-key certificates, the correct encoding for a CertBag
         <ReturnType>System.ReadOnlyMemory&lt;System.Byte&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the uninterpreted certificate contents of the CertSafeBag.</summary>
+        <value>The uninterpreted certificate contents of the CertSafeBag.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+For X.509 public-key certificates, the value of this property is a BER-encoded OCTET STRING value encompassing the certificate's DER encoded value.
+
+          ]]></format>
+        </remarks>
+        <altmember cref="M:System.Security.Cryptography.Pkcs.Pkcs12CertBag.GetCertificate()"/>
```

Comment:
```suggestion
        <altmember cref="M:System.Security.Cryptography.Pkcs.Pkcs12CertBag.GetCertificate"/>
```
This isn't appearing in the build, I suspect because the parens are unexpected.